op-guide/gc.md

```diff
@@ -0,0 +1,84 @@
+---
+title: TiDB Garbage Collection (GC)
+category: advanced
+---
+
+# TiDB Garbage Collection (GC)
+
+TiDB uses MVCC to control concurrency. When you update or delete data, the data is not deleted immediately but is saved for a period during which it can be read. Thus the write operation and the read operation are not mutually exclusive and it is possible to read the previous data.
```

Comment:
Maybe it's better to say "the **original** data is not deleted immediately but is **kept** for a while ....."

Reply:
"It is possible to read the **history versions** of the data". See op-guide/history-read.md